src/Microsoft.OData.Core/IWriterValidator.cs

```diff
@@ -15,10 +15,15 @@ namespace Microsoft.OData
     internal interface IWriterValidator
     {
         /// <summary>
-        /// Creates a DuplicatePropertyNameChecker instance.
+        /// Get a DuplicatePropertyNameChecker instance from the object pool.
         /// </summary>
         /// <returns>The created instance.</returns>

```

Comment:
nit: The DuplicatePropertyNameChecker instance. If it is just getting it and not creating it, i don't see why you are using the word 'created'